QUEST_LV_0100.tsv

```diff
@@ -1851,46 +1851,46 @@ QUEST_LV_0100_20150317_001850	{memo X}
 QUEST_LV_0100_20150317_001851	No antidotes yet?{nl}His pulse is fading.{nl}
 QUEST_LV_0100_20150317_001852	Thank you!{nl}Fortunately, he is breathing again.{nl}
 QUEST_LV_0100_20150317_001853	We'll rest a little more and return when he regains his strength.{nl}May the Goddess bless you.
-QUEST_LV_0100_20150317_001854	$The Revelator is here?! Thank the goddess..{nl}I am a historian, Rexipher. I am studying the Mausoleum.{nl}
-QUEST_LV_0100_20150317_001855	We already have the most of the answers.{nl}But to get the last answer, I need your help.
+QUEST_LV_0100_20150317_001854	$The Revelator is here?! Thank the Goddess.{nl}I am Rexipher, a historian. I am studying the Royal Mausoleum.{nl}
+QUEST_LV_0100_20150317_001855	$We already have most of the answers.{nl}But for the last answer, I need to borrow you, the Revelator's power.
```

Comment:
Hmm, your version sounds like Rexipher needs to temporarily borrow the Revelator's power. This is not the case. Rexipher needs the Revelator's aid, by using the powers. So, mine would fit better since Rexipher needs the Revelator himself in order to "borrow" the power. Get it? :stuck_out_tongue: